translations/README-en.md

```diff
@@ -0,0 +1,220 @@
+![Logotype](../logo.png)
```

Comment:
How about https://gitlocalize.com/ ? Keeping translation updated might be tricky

Reply:
Looks interesting. Can you Add the same comment to https://github.com/tlbootcamp/tlroadmap/pull/141, to leave this pr for one page translation and discuss the overall process in the other one?